test/runner/tob.ts

```diff
@@ -33,13 +36,20 @@ export async function file2Tob(filename: string): Promise<Tob> {
     filename,
     content,
     input: md2input(content),
+    transformOptions: md2transformOptions(content),
     parserOptions: md2parserOptions(content),
     printerOptions: md2printerOptions(content),
+    transform: md2cst(content),
```

Comment:
should be md2transform??

the functions(md2xxx) could be simplified to one function.

Reply:
Can you fix that? Get this working?